site/data/qasida/ya-nasim-al-ward.json

```
@@ -5,34 +5,61 @@
     },
     "verses": {
         "ara": [
-            ["يَا نَسِيمَ الوَرْدِ خَبِّرْ لِلرَشَا"],
-            ["لَم يَزِدْنِي الوِرْدُ إلَا عَطَشَا"],
+            ["لَا إِلَهَ إِلاَّ اللهْ اللهْ اللهْ"],
+            ["لَا إِلَهَ إِلَّا اللَّهُ جُدْ عَلَيْنَا"],
 
-            ["لِي حَبِيبٌ حُبُهُ وَسْطَ الحَشَا"],
-            ["إن يَشَأ يَمْشِي عَلَى خَدِّي مَشَا"],
+            ["يَا نَسِيْمَ الْوَرْدِ خَبِّرْ لِلرَّشَا"],
+            ["لَمْ يَزِدْنِي الْوِرْدُ إِلَّا عَطَشَا"],
 
-            ["قَولُهُ قَولِي وَقَولِي قَولُهُ"],
-            ["إن يَشَأ شِئتُ وَإن شِئتُ يَشَا"],
+            ["لِيْ حَبِيبٌ حُبُّهُ حَشْوُ الْحَشَا"],
+            ["إِنْ يَشَأْ يَمْشِي عَلَى خَدِّيْ مَشَى"],
 
-            ["رُوحُهُ رُوحِي وَرُوحِي رُوحُهُ"],
-            ["إن عَاشَا عِشْتُ وَإن عِشْتُ عَاشَا"]
+            ["(قَوْلُهُ قَوْلِي وَقَوْلِي قَوْلُهُ)"],
+            ["إِنْ يَشَأْ شِئْتُ وَإِنْ شِئْتُ يَشَا"],
+
+            ["رُوحُهُ رُوْحِيْ وَرُوْحِيْ رُوحُهُ"],
+            ["(إِنْ عَشَاعِشْتُ وَإِنْ عِشْتُ عَشَا)"],
+
+            ["(مِنْ وِشَاحِ الْحُسْنِ يَرْمِيْ رَمْشُهُ"],
+            ["يَا أَعَلُّ الْحُبُّ بَوَّاحاً وَشَى"],
+
+            ["سَلِمَتْ عَيْنَاهُ مِنْ مَقْتُولِهَا "],
+            ["وَحَمَى مَسْتُوْرَهَا دَاءُ الْعَشَى"],
+
+            ["مَا الَّذِي أَصْنَعُهُ فِي مُتْلِفِي"],
+            ["بَعْدَمَا فِيَّ مِنَ اللُّطْفِ فَشَى)"]
           ],
         "en": [
-            ["yā nasīm al-ward khabbir lil-rashā"],
-            ["lam yazidnī al-wirdu illā ʿaṭashā"],
+            ["lā ilāha illā Allāh Allāh Allāh"],
+            ["lā ilāha illā Allāhu jud ʿalaynā"],
```

```suggestion
            ["lā ilāha illā Allāh jud ʿalaynā"],
```
(Removed ending to match above comment)